pkg/network/infraconfigurators/bridge_test.go

```diff
@@ -189,4 +189,621 @@ var _ = Describe("Bridge infrastructure configurator", func() {
 			})
 		})
 	})
+
+	Context("prepare the pod networking infrastructure", func() {
+		const (
+			ifaceName     = "eth0"
+			launcherPID   = 1000
+			macStr        = "AF:B3:1F:78:2A:CA"
+			mtu           = 1000
+			queueCount    = uint32(0)
+			tapDeviceName = "tap0"
+		)
+
+		var (
+			bridgeConfigurator *BridgePodNetworkConfigurator
+			bridgeIPAddr       *netlink.Addr
+			bridgeIPStr        string
+			iface              *v1.Interface
+			inPodBridge        *netlink.Bridge
+			mac                net.HardwareAddr
+			podLink            *netlink.GenericLink
+			podIP              netlink.Addr
+			vmi                *v1.VirtualMachineInstance
+		)
+
+		BeforeEach(func() {
+			iface = v1.DefaultBridgeNetworkInterface()
+			vmi = newVMIWithBridgeInterface("default", "vm1")
+			mac, _ = net.ParseMAC(macStr)
```

Comment:
It would have been safer to do this at each test body, as not many need it. See the other cases where it is now referenced, which may not be the original intention.

Reply:
Done.